test/modules/standard/jama/TestMatrix.chpl

```diff
@@ -0,0 +1,403 @@
+use Jama;
+use Math;
+
+/** TestMatrix tests the functionality of the Jama Matrix class and associated decompositions.
+<P>
+Run the test from the command line using
+<BLOCKQUOTE><PRE><CODE>
```

Comment:
Would ya'll mind if this code module is renamed to something more recognizable? Maybe DenseLinearAlgebra or LinearAlgebra?